examples/python/braket.py

```diff
@@ -0,0 +1,124 @@
+# ## AWS via pytket
+
+# This notebook describes how to how to run a simple circuit using the `pytket-braket` backend from scratch.
+
+# ### Prerequisites
+
+# #### Python packages:
+
+# - `pytket`'s extension for Amazon Braket devices [pytket-braket](https://pypi.org/project/pytket-braket/)
+#     - `pip install pytket-braket`
+# - Amazon's braket package [amazon-braket-sdk](https://pypi.org/project/amazon-braket-sdk/)
+#     - `pip install amazon-braket-sdk`
```

Comment:
This is a dependency of pytket-braket, so it shouldn't be necessary to install separately.